src/uct/ib/rc/accel/rc_mlx5.inl

```diff
@@ -226,6 +230,14 @@ uct_rc_mlx5_iface_rx_mp_context_from_hash(uct_rc_mlx5_iface_common_t *iface,
 
     last = cqe->byte_cnt & htonl(UCT_IB_MLX5_MP_RQ_LAST_MSG_FLAG);
```

Comment:
1) can move `if` statement before setting `last` to optimize
or
2) can set `last` and `first` and check `if (ucs_likely(last && first)) {` instead

Reply:
yes, need to move, this was supposed to be set after the `if`

Reply:
@dmitrygx, fixed in-place since the change is small enough